code/modules/cargo/packs.dm

```diff
@@ -129,9 +129,10 @@
 
 /datum/supply_pack/emergency/metalfoam
 	name = "Metal Foam Grenade Crate"
-	desc = "Seal up those pesky hull breaches with 7 Metal Foam Grenades."
+	desc = "Seal up those pesky hull breaches with 14 Metal Foam Grenades."
 	cost = 1000
```

Comment:
Could at least double the cost of the crate to account for the doubled count of metal foam nades
```suggestion
	cost = 2000
```